language/en_us/alipay.php

```diff
@@ -2,6 +2,7 @@
 
 // Main
 $lang['Alipay.name'] = 'Alipay';
+$lang['Alipay.description'] = '';
```

Comment:
You should probably set some type of description relevant to this gateway, like the region it is mostly used in, what actions it supports, etc.

Reply:
Updated